src/containers/user-profile/comments-with-rating-block/CommentsWithRatingBlock.styles.ts

```diff
@@ -25,5 +25,8 @@ export const styles = {
     maxWidth: '250px',
     alignItems: 'center'
   } as SxProps<Theme>,
-  title: { typography: { xs: 'button', sm: 'h5', md: 'h4' } } as SxProps<Theme>
+  title: {
+    typography: { xs: 'button', sm: 'h5', md: 'h4' },
+    marginTop: '100px'
```

Comment:
it was decided to use a short form like `mt` and `pt`, etc for margin and padding In the project. so, it would be better to rewrite it to `mt`.